tests/output.rs

```diff
@@ -1113,3 +1120,67 @@ fn test_only_matching_multiline_overlapping_matches_starting_on_same_line() {
         "#,
     );
 }
+
+#[test]
+fn test_byte_offset() {
+    assert_sorted_output(
+        "rust_project",
+        r#"
+            $ tree-sitter-grep -q '(function_item) @function_item' -l rust --byte-offset
+            src/helpers.rs:1:0:pub fn helper() {}
+            src/stop.rs:1:0:fn stop_it() {}
+            src/lib.rs:3:14:pub fn add(left: usize, right: usize) -> usize {
```

Comment:
I'm seeing these tests fail in CI on Windows where it looks like the byte offset is bigger (by the row # - 1 bytes). That smells like "it's counting an additional `\r` for each line". But not totally clear on if/why the file contents (specifically the line endings) of the fixture files would be different on Windows?

So I think I'm going to rebase/re-target the other currently open PR against `master` and come back to this one

Reply:
Is it possible that git does some line ending magic to get you native line endings on whatever OS you're on?

Maybe, depending on your config, it looks like: https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings. Not sure what the default is.

Reply:
Nice ok I added `tests/fixtures/** eol=lf` to `.gitattributes` and it looks like it worked

That seems appropriate to me I guess I think that would just mean that someone editing something under `tests/fixtures/` on Windows wouldn't see "normal" Windows line endings?

Reply:
I guess that implies that we are no longer testing output when run against files with `\r\n` line endings though?

Reply:
Ok to avoid having to try and fully understand the implications of that, I just refined it to only impose that setting for the one fixtures directory for the `--byte-offset`-related tests (so all of the other existing tests will continue behaving the way they were before)

Reply:
> That seems appropriate to me I guess I think that would just mean that someone editing something under tests/fixtures/ on Windows wouldn't see "normal" Windows line endings?

Yeah but I think modern editors can deal with that fine these days, so it probably doesn't make much difference?